data/binding/mapbinding.go

```diff
@@ -89,12 +89,12 @@ type mapBase struct {
 // It will return nil if the key was not found.
 //
 // Since: 2.0.0
-func (b *mapBase) GetItem(key string) DataItem {
+func (b *mapBase) GetItem(key string) (DataItem, bool) {
```

Comment:
Could this be simplified to `return b.items[key]`?

Reply:
That doesn't work, believe it or not: `Not enough parameters to return` - I think the compiler does not see the optional second parameter so assumes it's missing and returns just the value...

Reply:
Huh, odd